gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -26,6 +31,7 @@ pub async fn run_webserver(
 
     // Authenticated, public routes used for gateway administration
     let admin_routes = Router::new()
+        // TODO: (@oleonardolima) Should the /info be really a POST ?
```

Comment:
@okjodom Was POST being used for any specific reason ? 🤔

Reply:
GET seems more appropriate